src/agent/agent.clj

```diff
@@ -594,6 +596,32 @@ fi")
                 :shell-stdout (:shell-stdout task)
                 :redacted true) nil])))
 
+(defn s3-upload [task]
+  (let [task-output (if (= (:shell-exit-code task) 0)
```

Comment:
Just as convention, maybe we put the `0` on the left side. This is because the comparator is placed generally in there.

As example, when we write tests, the `expected` value comes on the left side: `(is (= expected result))`. I can tell by the tests results, in which the label `expected` is assigned to the left operand.

Not a big deal, though...